app/views/shared/buttons/_ab_form.html.haml

```diff
@@ -1,25 +1,4 @@
 #ab_form
-  #policy_bar
-    - if session[:resolve_object].present?
-      - copied_target_class = session[:resolve_object][:new][:target_class]
-      - current_target_class =  @edit[:new][:target_class]
-      - if copied_target_class == current_target_class
-        = link_to({:action => "resolve", :button => "paste"},
```

Comment:
There's probably also a "resolve" action in the simulation controller that can be removed as well.  Might involve some more searching though.

Reply:
`def resolve
    custom_button_redirect = params[:button] == 'simulate' || params[:simulate] == 'simulate'
    assert_privileges(custom_button_redirect ? 'ab_button_simulate' : 'miq_ae_class_simulation')
    @explorer = true
    @breadcrumbs = []
    drop_breadcrumb(:name => _("Resolve"), :url => "/miq_ae_tools/resolve")
    @lastaction = "resolve"
    @right_cell_text = _("Simulation")

    case params[:button]
    when "throw", "retry" then resolve_button_throw
    when "copy"     then resolve_button_copy
    when "paste"    then resolve_button_paste
    when "simulate" then resolve_button_simulate
    else                 resolve_button_reset_or_none
    end
  end`
  
  I think this is what the resolve is referring to, so I should be able to remove the copy and paste cases